src/main/java/com/pablissimo/sonar/TsLintSensor.java

```diff
@@ -96,9 +96,28 @@ else if (config.getConfigFile() == null && config.getPathToTsConfig() == null) {
 
             String lowerFilePath = filePath.toLowerCase();
 
+
+
             if (!fileMap.containsKey(lowerFilePath)) {
                 LOG.warn("TsLint reported issues against a file that wasn't sent to it - will be ignored: " + lowerFilePath);
-                continue;
+
```

Comment:
Ha! Just started seeing the issue when building a sample app to demonstrate @NikitaEgorov's contribution of re-using the output of a previous tslint path - seems that tslint will report paths pretty much exactly how they were sent to it, so if supplied on the command line as relative paths then issues are reported against the relative paths.

Gives me an easy repro case, will take a look at a fix.